src/meta/types/Cargo.toml

```diff
@@ -15,10 +15,10 @@ common-datavalues = { path = "../../query/datavalues" }
 common-exception = { path = "../../common/exception" }
 common-storage = { path = "../../common/storage" }
 
-openraft = { git = "https://github.com/drmingdrmer/openraft", rev = "v0.7.2-alpha.4" }
+openraft = "0.7.2"
```

Comment:
What about keeping using the git source?

I'm afraid the next update will still be alpha, and it has to revert to using git source. 🤔

Reply:
> What about keeping using the git source?

Addressed. We are tracking the latest commit on `github.com/datafuselabs/openraft` branch `release-0.7`.